gamestonk_terminal/portfolio/port_menu.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+from matplotlib import pyplot as plt
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio import rh_api
+
+
+def print_port(show_login):
+
+    """ Print help """
+
+    print("\nPortfolio:")
+    print("   help          show this menu again")
+    print("   q             quit this menu, and shows back to main menu")
+    print("   quit          quit to abandon program")
+    print(" ")
+    print("Robinhood:")
+    print("")
+    if show_login:
+        print("   login         login")
+    print("   hold          look at current holdings")
```

Comment:
The way I see this in future would be, user goes into 'portfolio' and sees:
```
Portfolio
    help    
    q
    quit

    login

Current Broker(s): ?
    hold
    hist

Supported Brokers:
rh - Robinhood
tda -  TDAmeritrade
ig - IG
....
```

Then the user would do something along the lines:
```
(port)> login rh tda
```

which would show: `Current Broker(s): rh tda`

Then upon user doing 
```
(port)> hold
```
they would get the holdings of both of the stocks in both platforms, and the data would be merged.

Just a quick vision I had, let me know what you think

Reply:
Yeah I think that makes sense.

As an aside.  I liked the `hist` function, but it didn't get through the code spell.  Any chance that could be edited on the check side?    Or I can go with `history`